### PR TITLE
chore: re-organise monorepo

### PR DESCRIPTION
**Pull request by ctrlc03**
# Description

Cleanup monorepo structure by moving packages into common folders. 

Note: vercel will fail due to website path change (works when changing path to apps/website)
## Related issue(s)

re #1716 

## Confirmation

- [x] I have read and understand MACI's [contributor guidelines](https://maci.pse.dev/docs/contributing) and [code of conduct](https://maci.pse.dev/docs/contributing/code-of-conduct).
- [x] I have read and understand MACI's [GitHub processes](https://github.com/privacy-scaling-explorations/maci/discussions/847).
- [x] I have read and understand MACI's [testing guide](https://maci.pse.dev/docs/testing).

### CODE REVIEW

File: packages/cli/.gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ contractAddresses.old.json
 contractAddress.old 
 contractAddress.txt
 localState.json 
+zkeys
```
Comment (0xmad) [inline packages/cli/.gitignore:6]: Let's add new line at the end

**Review by 0xmad — COMMENTED** [review]
@ctrlc03 thanks, just one comment and let's merge it after #1700

File: packages/cli/.gitignore (hunk 1 above)
Comment (0xmad) [reply packages/cli/.gitignore:6]: It's still without new line

**Review by 0xmad — APPROVED** [review]
@ctrlc03 thanks!

**Comment by niconiahi** [comment]
great to see this pushed :)

**Pull request merged**